sig-cli/annual-report-2021.md

```diff
@@ -0,0 +1,71 @@
+        All of the meetings are recorded and available online, all the meeting invites are up-to-date and present in community calendar.
+
+* How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+    * As mentioned above, all the subprojects have a spot to report their progress during standups at the end of our biweekly calls. OWNERS files are up-to-date.
+
+* When was your last monthly community-wide update? (provide link to deck and/or recording)
+    * Last presentation was on April 16, 2020:
+    * [Slides](https://docs.google.com/presentation/d/1Y8SHFz6yyYS6rvRCgYUrSgF-moPk_YB6A-7Ykw5eWnU/edit#slide=id.g401c104a3c_0_)
+    * [Recording](https://youtu.be/Y3z2grPHRh4?t=415)
+
+## Membership
+
+* Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+    * Yes, all the lists are up-to-date.
+
+* How do you measure membership? By mailing list members, OWNERs, or something else?
+    * Anyone can be considered a SIG member if they join the Zoom calls or general discussions regularly.
+
+* How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+    * At our biweekly meetings, we track the progress of each feature the SIG is working on. Each feature has dev leads assigned to it. If a developer does not have time to complete the task we try to find someone else who has the bandwidth or we defer the feature to the next release.
+    * We periodically remove inactive members and invite members to join that we have seen active in SIG meetings and activities.
+
+* Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+    * Over the past month the leads take a bit more aggressive approach to closely monitor contributions and recognize the most active members by promoting them to reviewers and eventually approvers.
+
+* What programs do you participate in for new contributors?
+    * We try to feature “good-first-issue”s.
+    * As mentioned above, we give newcomers a spot to introduce at our biweekly calls.
+    * KubeCon updates.
+    * Individual/1-1 mentoring.
+    * In 2020 we participated in Google Season of Docs which resulted in [new documentation for kubectl and kustomize](https://kubectl.docs.kubernetes.io/).
+
+* Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?
+    * The group has contributors from multiple companies/affiliations.
+    * [28 companies contributed code in the last year](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20CLI&var-companies=All)
```

Comment:
do you welcome any more in certain areas? have a wishlist? this is the time to talk to end users

Reply:
We didn't dive that deep into stats, all newcomers are being treated equally.